lib/core/src/server/build-dev.js

```diff
@@ -151,12 +184,46 @@ Would you like to run Storybook on port ${port} instead?`,
   });
 
   try {
-    const [stats] = await Promise.all([webpackValid, serverListening]);
+    const [stats, updateInfo] = await Promise.all([webpackValid, updateCheck(), serverListening]);
     const proto = program.https ? 'https' : 'http';
     const address = `${proto}://${program.host || 'localhost'}:${port}/`;
     const networkAddress = `${proto}://${ip.address()}:${port}/`;
-    logger.info(`Storybook started on => ${chalk.cyan(address)}`);
-    logger.info(`Available on the network at => ${chalk.cyan(networkAddress)}\n`);
+
+    const updateMessage =
+      updateInfo.success && semver.lt(packageJson.version, updateInfo.data.latest.version)
+        ? stripIndents`
+          ${chalk.hex('#F3AD38')(
+            `A new version (${chalk.bold(updateInfo.data.latest.version)}) is available!`
+          )}
+          ${chalk.gray(updateInfo.data.latest.info.plain)}
+
+          ${chalk.gray('Read full changelog here:')}
+          ${chalk.gray.underline('https://git.io/fxc61')}
```

Comment:
What is this URL?

Reply:
short version of a direct link to our changelog.md on master

Reply:
as it's printed in the terminal I wanted to have a url short enough to display on 1 line